CONTRIBUTING.md

```diff
@@ -0,0 +1,221 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+
+You can contribute in one of the following ways:
+
+1. Submit a bug report or a feature
+   request in [ our public Jira ](https://jira.percona.com/projects/PS/issues).
+2. Submit a pull request (PR) with the code patch.
+3. Contribute to [the documentation](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md)
+4. Reach us on our [ Forums ](https://forums.percona.com/c/mysql-mariadb)
+
+This document describes the workflow for submitting pull requests.
+By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). Thank
+you for helping us enhance Percona Server for MySQL.
+
+## Why Your Contribution Matters?
+
+Main driving force of open-source project is community. We, as developers team, highly appreciate any contributions.
+
+It may be advisable to initially submit bug fixes to the upstream project that will eventually become part of the next
+release of Percona. Alternatively, Percona provides expedited response times, transparent development processes, and
+streamlined contribution acceptance within a public repository.
+
+## Creating a Jira Issue
+
+The creation of a Jira issue is _mandatory_. A comprehensive description of the bug or feature is essential. Thorough
+testing of the solution, inclusion of pertinent information in the release notes, and other relevant details are highly
+recommended. Additionally, linking the pull request to the corresponding issue with a detailed description will
+facilitate the development process.
+
+To create Jira issue, please open the  [ LINK ](https://perconadev.atlassian.net/jira/software/c/projects/PS/issues) and
+authenticate with your favorite social network account (Google, Facebook, etc.). Then follow the instructions and verify
+your e-mail. If Jira starts to redirect you back and forth (their bug, not ours), just open a new browser window with
+the link above.
+
+Now you're ready to an create issue. Find the big "Create" button in the top menu and press it. Choose issue type from
+the second drop-down, saying it is a Bug, an Improvement, or a New feature. Fill in all the required fields below. Don't
+spare the letters for the description field. Accurate and detailed description is a huge first step to the solution. The
+rest of the fields are mostly for internal use; the core team developers will fill it later.
+
+## Working with the Source Code
```

Comment:
This doesn't fit the flow. 
We list
Submit a Jira
then 
Submit a pull request

There's nothing about working with source code. Why would a user do this? If it's important, we should list it in the items menu.

Reply:
I structured this part of the document and added some explanations. Please  check again.